### PR TITLE
fix: detach when DRD is detached

### LINKED ISSUE #38: Properties panel does not detach when decision table is opened via API
__Describe the Bug__

When I switch view via `modeler.open(decisionTableView)`, the properties panel is not detached. [It reacts only on drilldown event](https://github.com/bpmn-io/dmn-js-properties-panel/blob/master/src/render/DmnPropertiesPanelRenderer.js#L49).

__Steps to Reproduce__

1. Create a modeler
2. Run `modeler.open(modeler.getViews().find(view => view.type === 'decisionTable')`
3. now this happens

__Expected Behavior__

Properties panel should detach.

__Environment__

- Host (Browser/Node version), if applicable: [e.g. MS Edge 18, Chrome 69, Node 10 LTS]
- OS: [e.g. Windows 7]
- Library version: 1.0.0-alpha.1

---

Discovered when I tried to re-create https://github.com/bpmn-io/dmn-js-properties-panel/issues/37 in a test case.

### PR DESCRIPTION
**Pull request by barmac**
Closes #38

### CODE REVIEW

File: src/render/DmnPropertiesPanelRenderer.js [hunk 1]
```diff
@@ -46,7 +46,11 @@ export default class DmnPropertiesPanelRenderer {
       this._render(element);
     });
 
-    eventBus.on('drillDown.click', (event) => {
+    eventBus.on('detach', (event) => {
+      this.detach();
+    });
+
+    dmnjs.on('detach', () => {
```
Comment (pinussilvestrus) [inline src/render/DmnPropertiesPanelRenderer.js:53]: Are these two different events that are fired?

**Comment by barmac** [comment]
Yes. The event on event bus is fired when the viewer is changed while the one on dmn-js is fired when the manager is detached.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

File: src/render/DmnPropertiesPanelRenderer.js (hunk 1 above)
Comment (barmac) [reply src/render/DmnPropertiesPanelRenderer.js:53]: Yes. The event on event bus is fired when the viewer is changed while the one on dmn-js is fired when the manager is detached.

File: test/spec/DmnPropertiesPanelRenderer.spec.js [hunk 2]
```diff
@@ -135,17 +135,47 @@ describe('<DmnPropertiesPanelRenderer>', function() {
...
+      expect(domQuery('.bio-properties-panel-container', propertiesContainer)).to.not.exist;
+    });
+
+
+    it('should detach when view switched to decision table via modeler.open', async function() {
+
+      // given
+      const { modeler } = await createModeler(diagramXml);
+      const decisionTableView = modeler.getViews().find(view => view.type === 'decisionTable');
+
```
Comment (pinussilvestrus) [inline test/spec/DmnPropertiesPanelRenderer.spec.js:159]:
```suggestion

expect(domQuery('.bio-properties-panel-container', propertiesContainer)).to.exist;

```

Maybe, just to double-check? The same would go for the other tests.

**Review by pinussilvestrus — APPROVED** [review]
🚀
